lib/sorcery/controller/submodules/external.rb

```diff
@@ -40,7 +40,7 @@ def external_providers=(providers)
                 providers.each do |name|
                   class_eval <<-RUBY, __FILE__, __LINE__ + 1
                     def self.#{name}
-                      @#{name} ||= Sorcery::Providers.const_get('#{name}'.to_s.classify).new
+                      @#{name} ||= Sorcery::Providers.const_get('#{name}'.to_s.camelcase).new
```

Comment:
@epdejager I was worried this would break the original reason for switching to classify, but some quick testing in rails console alleviated those concerns. Good fix!

Reply:
Yes indeed, I changed it back to `capitalize` first but then realised it broke the spec for providers with underscores in their names. `camelcase` seems to do the trick for both 🤞